src/client/theme-default/components/LastUpdated.vue

```diff
@@ -16,11 +16,16 @@ const prefix = computed(() => {
 })
 
 const datetime = ref('')
+
 onMounted(() => {
   // locale string might be different based on end user
   // and will lead to potential hydration mismatch if calculated at build time
   datetime.value = new Date(page.value.lastUpdated).toLocaleString('en-US')
 })
+
+onUpdated(() => {
+  datetime.value = new Date(page.value.lastUpdated).toLocaleString('en-US')
```

Comment:
Why is this necessary? It's already done in onMounted

Reply:
When route changing, without `onUpdated` hook leading to the same last updated time on every page.

Cause I'm not familar with Vue, I don't know why it happened.